CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ All notable changes to this extension will be documented in this file.
 - React to websocket events pushed from sidecar instead of polling for some connection change
   monitoring.
 
+### Fixed
+
+- Extension/sidecar logs and the support zip can now be downloaded on Windows as expected.
```

Comment:
"downloaded"? Howsabout "generated" ?

Reply:
Ah I remember us talking about this in the original PR. Changing to "saving" since that's what... half of the mentions were using. https://github.com/confluentinc/vscode/pull/938/commits/46050ece821f9f8f6d917a3f761a67e38477b294